DOCS-969: Rename radius_cluster_segmenter (#1628)
viamrobotics/docs: docs/services/vision/segmentation.md

The types of segmenters supported are:

> - [**Obstacles Pointcloud (`obstacles_pointcloud`)**](#configure-a-obstacles_pointcloud): A segmenter that identifies well separated objects above a flat plane.

Author: ```suggestion
- [**Obstacles Pointcloud (`obstacles_pointcloud`)**](#configure-an-obstacles_pointcloud): A segmenter that identifies well separated objects above a flat plane.
```

    "min_points_in_plane": <integer>,
    "min_points_in_segment": <integer>,
    "clustering_radius_mm": <number>,
    "mean_k_filtering": <integer>,
    "max_dist_from_plane_mm": <integer>,

>     "ground_plane_normal_vec": <integer>,

Author: ```suggestion
        "ground_plane_normal_vec": {
          "x": <integer>,
          "y": <integer>,
          "z": <integer>
        },
```

>     "ground_angle_tolerance_degs": <>

Author: ```suggestion
    "ground_angle_tolerance_degs": <number>
```

| `clustering_radius_mm` | _Required_ | A float that specifies how far apart points can be (in units of mm) in order to be considered part of the same object. A small clustering radius will more likely split different parts of a large object into distinct objects. A large clustering radius may aggregate closely spaced objects into one object. 3.0 is a decent starting value. |
| `mean_k_filtering` | _Optional_ | An integer parameter used in [a subroutine to eliminate the noise in the point clouds](https://pcl.readthedocs.io/projects/tutorials/en/latest/statistical_outlier.html). It should be set to be 5-10% of the number of min_points_in_segment. Start with 5% and go up if objects are still too noisy. If you don’t want to use the filtering, set the number to 0 or less. |
| `max_dist_from_plane_mm` | Optional | A float that determines how much area above and below an ideal ground plane should count as the plane for which points are removed. For fields with tall grass, this should be a high number. The default value is 100 mm. |

> | `ground_plane_normal_vec` | Optional | A `(x,y,z)` vector that represents the normal vector of the ground plane. Different cameras have different coordinate systems. For example, a lidar's ground plane will point in the `+z` direction `(0, 0, 1)`. On the other hand, the intel realsense `+z` direction points out of the camera lens, and its ground plane is in the negative y direction `(0, -1, 0)`. The default value is `(0, 0, 1)`. |

Reviewer: Yes, these are the cartesian coordinates r3.vector. This is the arrow that points perpendicular to the ground plane.

> | `ground_angle_tolerance_degs` | Optional | A float that determines how strictly the found ground plane should match the `ground_plane_normal_vec`. For example, even if the ideal ground plane is purely flat, a rover may encounter slopes and hills. The algorithm should find a ground plane even if the found plane is at a slant, up to a certain point. The default value is 30 degrees. |

Reviewer 2: ```suggestion
| `ground_angle_tolerance_degs` | Optional | A float that determines how strictly the found ground plane should match the `ground_plane_normal_vec`. For example, even if the ideal ground plane is purely flat, a rover may encounter slopes and hills. The algorithm should find a ground plane even if the found plane is at a slant, up to a certain point. <br> Default: `30.00` </br> |
```

(Opt. if we are still doing this with andrew's table format updates you could do this the same way in the column above!)
